src/pipeline_migration/actions/modify/generic.py

```diff
@@ -175,17 +185,17 @@ def register_cli(subparser) -> None:
     subparser_replace.add_argument(
         "yaml_path",
         help=(
-            "YAML path (in YAML format). Must point to a sequence or a map item. "
+            "YAML path (in YAML format). Can point to a scalar value. "
```

Comment:
"Can point to a scalar" makes sense in the context of this PR, but probably won't make sense on its own in the help text. It just raises the question: "Wait, _only_ to a scalar"?

IMO we should remove it.

Reply:
updated